l10n_br_account_product/models/account_invoice.py

```diff
@@ -861,8 +861,10 @@ def _validate_taxes(self, values):
         freight_value = values.get('freight_value', 0.0) or self.freight_value
         other_costs_value = values.get(
             'other_costs_value', 0.0) or self.other_costs_value
-        tax_ids = values.get('invoice_line_tax_id', [[6, 0, []]])[
-            0][2] or self.invoice_line_tax_id.ids
+        tax_ids = []
+        if values.get('invoice_line_tax_id'):
+            tax_ids = values.get('invoice_line_tax_id', [[6, 0, []]])[
+                0][2] or self.invoice_line_tax_id.ids
```

Comment:
@renatonlima is it normal to touch l10n_br_account_product in this PR?
I mean is it on purpose or by mistake?